updater/Gemfile

```diff
@@ -27,12 +27,15 @@ gem "sentry-raven", "~> 3.1"
 gem "terminal-table", "~> 3.0.2"
 
 group :test do
-  gem "debug", "~> 1.6.3"
-  gem "rspec", "~> 3.12"
-  gem "rubocop", "~> 1.37.1"
-  gem "rubocop-performance", "~> 1.15.0"
-  gem "vcr", "~> 6.1"
-  gem "webmock", "~> 3.18"
+  common_gemspec = File.expand_path("../common/dependabot-common.gemspec", __dir__)
+
+  Dir.chdir(File.dirname(common_gemspec)) do
+    Gem::Specification.load(common_gemspec).development_dependencies.each do |dep|
+      next unless %(debug rspec rubocop rubocop-performance vcr webmock).include?(dep.name)
```

Comment:
it's really easy to overlook that this is _the_ list of gems pulled in here.

A stupid simple change that would make it visually stand out in the source is to make it a multi-line list:
```
debug
rspec
rubocop
rubocop-performance
vcr
webmock
```

dunno if they need commas, but I'm sure you'll know the answer to that. 😄

Reply:
Thanks, updated!